### ISSUE TITLE
Negative review count

### ISSUE THREAD

**Issue by gaul** [helloworld1/AnyMemo#463]
See top left status line:

![screenshot_20181225-235754](https://user-images.githubusercontent.com/848247/50461425-2c233e00-0933-11e9-8bae-2e6d01a4b03f.jpg)

**Comment by helloworld1** [comment]
If you can email me the db file, I can investigate further.

**Comment by gaul** [comment]
I suspect that this happens when a card becomes eligible for review during a long-lived study session.

**Comment by helloworld1** [comment]
Yes It could be. The count does not reflect the status of the db. It just increments / decrements after you grade a card. However the actual card you reviewed is accurately reflecting the current time not the beginning of the study session. 

If you exit the study screen and come back, and the number becomes correct, it would be the reason you suspected.
